source/_docs/z-wave.markdown

```diff
@@ -180,6 +180,36 @@ automation:
         scene_id: 11
 ```
 
+Some devices (like the HomeSeer wall switches) allow you to do things like double, and triple click the up and down buttons and fire an event.  This does support the OpenZwave CentralScene handler.  This is an example of double clicking the on/up button:
+
+```yaml
+# Example configuration.yaml automation entry
+automation
+  - alias: 'Dining room dimmer - double tap up'
+    trigger:
+      - event_data:
+          entity_id: light.diningroomcans_level
+          scene_id: 1
+          scene_data: 3
+        event_type: zwave.scene_activated
+        platform: event
+    action:
```

Comment:
Drop the action portion here to keep it consistent with the rest of the examples. It'll just be an example of the trigger portion.